web/src/lib/utils.ts

```diff
@@ -0,0 +1,30 @@
+import {
+	type Chain,
+	type ChainID,
+	type FungibleToken,
+	type TokenSymbol
+} from '@chainsafe/gopher-sdk';
+
+const noNetworkFound: Chain = {
+	chainID: 0,
+	chainType: 'evm',
+	name: 'none',
+	logoURI: 'https://static.thenounproject.com/png/75231-200.png',
+	rpcurls: []
+};
```

Comment:
same as last comment, just an observation, maybe `satisfies` would be more appropriate here

```suggestion
const noNetworkFound = {
	chainID: 0,
	chainType: 'evm',
	name: 'none',
	logoURI: 'https://static.thenounproject.com/png/75231-200.png',
	rpcurls: []
} satisfies Chain;
```